mne/io/fiff/tests/test_raw_fiff.py

```diff
@@ -411,22 +411,22 @@ def test_concatenate_raws_bads_order():
     # Bad channel mismatch raises
     raw2 = raw1.copy()
     raw2.info["bads"] = ["0", "2"]
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match="must match"):
         concatenate_raws([raw0, raw2])
 
     # Type mismatch raises
     epochs1 = make_fixed_length_epochs(raw1)
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match="type must match"):
         concatenate_raws([raw0, epochs1.load_data()])
 
     # Sample rate mismatch
     raw3 = _create_toy_data(sfreq=500)
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match="must match"):
         concatenate_raws([raw0, raw3])
 
     # Number of channels mismatch
     raw4 = _create_toy_data(n_channels=4)
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match="must match"):
```

Comment:
```suggestion
    with pytest.raises(ValueError, match="order must match"):
```

Reply:
This also does not work @drammock. The error message is 
```Python
ValueError: raws[1].info['nchan'] must match
```
since the number of channels does not match.

Reply:
oops!  I was incorrectly inferring which error message this was supposed to raise.  I still think the `match` value needs to change though: apparently we have at least 3 errors that are possible to encounter in this context, and all of them contain the string `must match` so we need `match` to be more specific about which one we expect.